pkgs/by-name/ar/artalk/package.nix

```
@@ -31,11 +33,25 @@ buildGoModule rec {
     "-X github.com/ArtalkJS/Artalk/internal/config.Version=${version}"
     "-X github.com/ArtalkJS/Artalk/internal/config.CommitHash=${version}"
   ];
+
+  nativeBuildInputs = [ installShellFiles ];
+
   preBuild = ''
     tar -xzf ${web}
     cp -r ./artalk_ui/* ./public
   '';
 
+  postInstall =
+    ''
+      mv $out/bin/Artalk $out/bin/artalk
```

```suggestion
      ln -s $out/bin/Artalk $out/bin/artalk
```
that's probably better to preserve backwards combability and maybe match upstream docs